common_utils.py

```diff
@@ -221,21 +221,24 @@ def dragMoveEvent(self, event):
     def dropEvent(self, event):
         data = event.mimeData()
         mime = "text/uri-list"
-        url = str(data.data(mime))
-        path = urlparse(url).path.strip()
-        scheme = urlparse(url).scheme
+        url = urlparse(str(data.data(mime), encoding='utf-8').rstrip())
+        path = url.path.strip()
+        scheme = url.scheme
```

Comment:
In my testing, if I didn't encode it to utf-8, the urlparser wasn't able to parse a file path